addons/github/static/githubFangornConfig.js

```diff
@@ -300,6 +300,7 @@ var _githubItemButtons = {
 
 function changeBranch(item, ref){
     item.data.branch = ref;
+    item.data.urls.repo = item.data.urls.repo.substring(0, item.data.urls.repo.lastIndexOf('/tree/') + 1) + 'tree/' + ref;
```

Comment:
Could you break this up into multiple lines for readability, please? Probably two more lines would be the right number (one for getting the index, one for grabbing the substring).